kubernetes/apps/finesse/finesse-backend-deployment.yml

```diff
@@ -53,7 +45,7 @@ spec:
       serviceAccountName: secrets-reader
       containers:
       - name: finesse-backend
-        image: gcr.io/spartan-rhino-408115/finesse-backend:latest
+        image: ghcr.io/ai-cfia/finesse-backend:98f702dddef8f5d46668794b65920c9ce38b8034
```

Comment:
don't we have additional more human-readable tags being created by the build process?

Reply:
we do

Reply:
should we not be using them here?

Reply:
Yes, I changed it to the PR number since the branch name won't follow the image digest when we merge it into the main branch. Also, using the PR number makes it easy to track which PR. Anyone can go to the pull request tab and enter the pull request number to see what changes were made.